fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx

```diff
@@ -0,0 +1,165 @@
+
+Open the `docs.yml` file and locate the `url`, which looks like this:
+
+```yml
+instances:
+  - url: petstore-ferndef.docs.buildwithfern.com
+```
+
+Replace `petstore-ferndef` with your own organization's name. Use only alphanumeric characters, hyphens, and underscores. Do not use spaces, and leave the rest of the URL (`.docs.buildwithfern.com`) unchanged.
+
+### Step 4: Install the Fern CLI tool
+
+Install the Fern CLI tool globally by running:
+
+```shell
+npm install -g fern-api
+```
+
+<Callout intent='info'>
+The above CLI command is a global command that you can run from any location. The `fern` commands in the following steps must be run from within your repository.
+</Callout>
+
+### Step 5: Generate your documentation
+
+Run the following command:
+
+```shell
+fern generate --docs
+```
+
+When prompted, log into and connect your GitHub account.
+
+Once the documentation is generated, Fern displays the URL where you can view the published documentation. For example:
+
+```shell
+┌─
+│ ✓  petstore-ferndef.docs.buildwithfern.com
+└─
+```
+
+Instead of `petstore-ferndef`, you should see the organization name that you added to `docs.yml` in Step 3. 
+
+### Step 6: Customize your documentation
+
+You must run `fern generate --docs` after any modifications to re-generate and publish your documentation site.
+
+To preview updates to your documentation before publishing changes, run `fern generate --docs --preview`.
+
+To modify the API Reference:
+
+- Update or replace the Fern Definition files in the `definition/` folder. See our documentation on [how to define your API with Fern Definition](define-your-api/ferndef/overview).
+
+To modify the other docs pages or add new pages:
+
+- Update the files located in the `docs/pages/` folder, such as `welcome.mdx`. See our documentation on [how to write content with Markdown and MDX](/fern-docs/content/write-markdown).
+
+To configure site navigation, styles, and see other configuration options:
+
+- See [Fern Docs Configuration Overview](/fern-docs/config/overview).
+
+To learn about Fern's built-in component library you can use in Markdown:
+
+- See the [Component Library](/fern-docs/content/component-library/).
+
+### Step 7: Set up a custom domain
+
+If you wish to use a custom subdomain like `https://docs.YOUR_ORGANIZATION.COM` or a subpath like `https://YOUR_ORGANIZATION.COM/docs`, you can subscribe to the [Starter plan](https://buildwithfern.com/pricing). Once subscribed, update `docs.yml` with the custom domain configuration, replacing YOUR_ORGANIZATION with your own:
+
+<CodeBlocks>
+    <CodeBlock title='Custom subdomain'>
+    ```yaml
+    - url: YOUR_ORGANIZATION.docs.buildwithfern.com
+      custom-domain: docs.YOUR_ORGANIZATION.COM
+    ```
+    </CodeBlock>
+    <CodeBlock title='Custom subpath'>
+    ```yaml
+    - url: YOUR_ORGANIZATION.docs.buildwithfern.com

```

Comment:
This should be YOUR_ORGANIZATION.docs.buildwithfern.com/docs

Reply:
Do you mean `YOUR_ORGANIZATION.buildwithfern.com/docs`?
Or still keep the subdomain `docs` before `buildwithfern`?

Reply:
I believe it is: YOUR_ORGANIZATION.docs.buildwithfern.com/<subpath>

The first `.docs.` cannot be changed. The second one can be whatever the customer wants the subpath to be (e.g., /docs)

Reply:
@minaelee 

Talked to Andrew:

for custom subpath it is

- url: <subdomain>.docs.buildwithfern.com/<base_url>

We recommend folks to make the subdomain their organization name, but this isn't required.
When using a subpath, there is no `custom-domain` key used.

Reply:
It seems like this info is better suited as updates to the `Custom domains` page (https://github.com/fern-api/fern/blob/main/fern/docs/pages/fern-docs/config/branding/custom-domain.mdx).

I can update there and swap out the info in both the docs-quickstarts with a link to the `Custom domains` page. How does that sound?

Reply:
e.g.:

### Step 7: Set up a custom domain, subdomain, or subpath

If you wish to use a custom domain (such as `https://example.com`), subdomain (such as `https://docs.example.com`), or subpath (such as `https://example.docs.buildwithfern.com/learn`), visit the [Custom domain](/fern-docs/config/branding/custom-domain) page.
```

Reply:
Love it! Let's do that. 

(I was mixed up in where I was leaving my comment.)